vite-project/src/App.css

```diff
@@ -40,3 +40,14 @@
 .read-the-docs {
   color: #888;
 }
+
+#container {
+  display: flex;
+  width: 1350px;
+  flex-direction: row;
+}
+
+#formCar {
+  margin-right: 300px;
+  margin-left: 120px;
+}
```

Comment:
Using selectors by id to declare styles is not convenient for reusability, because in this case, styles can be applied only to one element with a given id. In your project, this is not a problem, but when working on larger projects, it is more convenient to have certain classes that can be used several times, avoiding code duplication